queries.sql

```diff
@@ -1,34 +1,66 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-
```

Comment:
According to the project requirements, the following query should be present, please add it.

<img width="575" alt="Screen Shot 2022-08-26 at 8 55 49" src="https://user-images.githubusercontent.com/82426682/186919908-f7b600d3-2745-472e-9701-adfc0db8d04f.png">